Vagrantfile

```diff
@@ -75,6 +75,7 @@ Vagrant.configure("2") do |config|
         python3 \
         python3-dev \
         python3-pip \
+        python3-venv \
```

Comment:
Not actually a requirement on Debian.

This package provides the `pyvenv` binary but you can still run `python3 -m venv` without it.

(Tested on my home desktop, which runs Debian 10 Buster).

Reply:
Removed

Reply:
... not actually removed?